library/core/src/ops/range.rs

```diff
@@ -52,7 +52,7 @@ impl fmt::Debug for RangeFull {
 /// A (half-open) range bounded inclusively below and exclusively above
 /// (`start..end`).
```

Comment:
This sentence is confusing to me.

Reply:
Hm, looks pretty clear to me. What makes it confusing to you? And does my proposal make it worse or is it unrelated?

Reply:
I would rather leave this the way it is for now since only the first paragraph is shown in the item listing for a module and it's helpful to see what syntax this represents in the summary:

![image](https://user-images.githubusercontent.com/37223377/96385852-66cf1980-114b-11eb-8c67-c69fa6be9018.png)

Reply:
Ah, yes, that's a very good point. Agreed, that mini-example should stay.